src/parser/jobs/dnc/modules/DirtyDancing.tsx

```diff
@@ -156,7 +156,7 @@ export class DirtyDancing extends ActionWindow {
 		const statusTime = Math.max(this.statuses.getUptime(statusKey, this.actors.friends) - this.downtime.getDowntime(), 0)
 		const uptime = Math.max(this.parser.currentDuration - this.downtime.getDowntime(), 0)
 
-		return (statusTime / uptime) * 100
+		return Math.min((statusTime / uptime) * 100, 100)
```

Comment:
We intentionally don't do clamps like this, so that the higher number shows in development when the calculations go haywire (and the green bar number before you "unroll" the checklist item to view the total does clamp in production).

What's going on under the hood here is that FFLogs changed their definition of "0.00s" to be the start of combat to level out some speed racing hijinx people were doing, and they're now including some pre-pull events along with that change, which has changed the definition of "currentDuration" because now there's a little more event time in the pull than combat time.  But it's most visible with Dance Partner, since that's generally set completely external to the pull and thus should "always" get the over 100% total of including the pre-pull event timing.